app/bundles/PointBundle/Model/PointModel.php

```diff
@@ -197,9 +197,11 @@ public function getPointActions()
      */
     public function triggerAction($type, $eventDetails = null, $typeId = null, Lead $lead = null)
     {
-        //only trigger actions for anonymous users
-        if (!$this->security->isAnonymous()) {
-            return;
+        if($type != "api.call") {
+        	//only trigger actions for anonymous users
+        	if (!$this->security->isAnonymous()) {
+            	return;
+        	}
```

Comment:
@npracht again, the indentation is wrong, if needs a space. Also, it'd be nice to use single quotes for string values as Mautic code uses them everywhere else.